griptape/drivers/embedding/amazon_bedrock_embedding_driver.py

```diff
@@ -0,0 +1,47 @@
+import json
+import os
+import boto3
+from attr import define, field, Factory
+from griptape.drivers import BaseEmbeddingDriver
+from griptape.tokenizers import AmazonBedrockTokenizer
+
+
+@define
+class AmazonBedrockEmbeddingDriver(BaseEmbeddingDriver):
+    DEFAULT_MODEL = "amazon.titan-e1t-medium"
+
+    dimensions: int = field(default=1536, kw_only=True)
+    model: str = field(default=DEFAULT_MODEL, kw_only=True)
+    tokenizer: AmazonBedrockTokenizer = field(
+        default=Factory(
+            lambda self: AmazonBedrockTokenizer(model=self.model), takes_self=True
+        ),
+        kw_only=True,
+    )
+    session: boto3.Session = field(
+        default=Factory(lambda: boto3.Session()), kw_only=True
+    )
+    content_type: str = field(default="application/json", kw_only=True)
+    accept: str = field(default="application/json", kw_only=True)
```

Comment:
Could it ever be anything else?

Reply:
Yeah, I think removing it is fine. We can always introduce it later if some model requires something custom. And in that case, I imagine, it would live in the model driver.